src/lib/swapper/swappers/LifiSwapper/utils/getUnsignedTx/getUnsignedTx.ts

```diff
@@ -43,7 +44,9 @@ const createBuildSendApiTxInput = async (
     from,
     chainSpecific: {
       gasPrice: gasPrice.toString(),
-      gasLimit: gasLimit.toString(),
+      // Conform to Li.Fi SDK's gasLimit buffer
+      // https://github.com/lifinance/sdk/blob/ca2c5d6fa789346d415f1d55a4b9bde15c683385/src/allowance/utils.ts#L60
+      gasLimit: bnOrZero(gasLimit?.toString()).times(1.25).toString(),
```

Comment:
Also wrong and needs to be discriminated on the `lifiStep` for approvals only

Reply:
Remove me as well - not really related to approvals, though it may or may not when multi-hop is in